sdk/src/Services/DynamoDBv2/Custom/DataModel/IDynamoDBContextBuilder.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using System;
+
+namespace Amazon.DynamoDBv2.DataModel
+{
+    /// <summary>
+    /// Interface for a builder that constructs a <see cref="DynamoDBContext"/>
```

Comment:
Add would suggest giving more context for users upgrading about how this newer approach disables the `DescribeTable` mechanism. And explain how the can use the `ConfigureContext` method to revert to pervious behavior if they need it.

Reply:
Added more docs in the interface and inheritor.

Reply:
Add a line that says alternatively to attributes on the class you can register the table definition using the `RegisterTableDefinition` method on the context object.

Reply:
Done